packages/colors/README.md

```diff
@@ -0,0 +1,11 @@
+# `colors`
```

Comment:
this header usually name full package name which in this case is `@livechat/design-system-colors`

Reply:
renamed